client/state/sites/selectors/is-jetpack-site.js

```diff
@@ -1,18 +1,36 @@
 import getRawSite from 'calypso/state/selectors/get-raw-site';
+import isAtomicSite from 'calypso/state/selectors/is-site-automated-transfer';
+import { AppState } from 'calypso/types';
```

Comment:
You don't need to actually import this type because it's not being used (this is not a TypeScript file). You can change the `@param` line below to import just the type by doing:

```
@param {import('calypso/types').AppState} state Global state tree
```

Reply:
Oh yeah! Actually, I first converted it to TypeScript and imported that type but there was some TS error elsewhere that I didn't want to spend time fixing. So I converted it back to JS and that import was left there.
Updated as per your suggestion. Thanks.